game.js

```diff
@@ -398,8 +398,8 @@ class Game {
                 warnAndResign("place_free_handicap returned wrong number of handicap stones, resigning.");
                 return;
             }
-            for (let i in moves)                     // Sanity check
-                if (moves[i].pass || moves[i].x < 0) {
+            for (let e in moves)                     // Sanity check
+                if (moves[e].pass || moves[e].x < 0) {
```

Comment:
You would want to replace this with

```
for (let e of moves) 
    if (e.pass || e.x < 0) {
```
Else you if you keep it at `in`, it makes more sense to also keep `i` for index.

Reply:
Actually, i would use `move` instead of `e` to make it even more clear.

Reply:
@Dorus 

i didnt know, my bad !!

ok i'll revert it then :+1:

Reply:
i'm not sure move makes it clearer than i (index)

since we are referring to an index, i'd rather use `i` i think, 
but i dont know :+1:

Reply:
> i'm not sure move makes it clearer than i (index)

I literally gave you the code how to replace `let i in moves` with `let move of moves`.

Anyway not changing it is fine too, either way works. It's all just code cleanup after all.

Reply:
@Dorus 

i really dont know a lot about these things, i didnt want to ignore your advice, sorry :+1: 

but as i see things, `i` is indeed an index contrary to what i thought, so using `i` was actually fine, so i reverted to it

your code suggests not a renaming but a modification
it is indeed interesting to refer to `e` rather than `move[i]`

but since i dont know a lot about this part of the code, i think its better that i avoid modifying it
maybe in some more general code cleanup it will be meaningful to change this part as well as all other similar parts in the same commit :+1:

Reply:
Yes, if you change `i in` to `move of`, you change what the variable is. It goes from the index to the element. So instead of `moves[i]` later on to get the element, you can use `move` directly.

Anyway as i said, reverting is is perfectly fine too. I was just adding information for you to have a complete picture.